### DISCUSSION TITLE
Can't Create A Character

### DISCUSSION THREAD

**Discussion by dell-sackett** [chromiecraft/chromiecraft#5874] [General]
I'm new and would love to play CC but every time I try to make a character I'm just told that character creation is disabled. Anyone know why?

**Comment by 55Honey** [comment]
Hi @lospheris 
I take it you are trying to create a Deathknight. These are not available until we venture into Northrend.
Feel free to catch up with our planned progression: https://www.chromiecraft.com/en/progression/

Regards

**Reply by dell-sackett** [reply]
Yup. I'm an idiot and totally missed that. Thank you very much for the help!
